src/sentry_uuid.h

```diff
@@ -3,11 +3,22 @@
 
 #include "sentry_boot.h"
 
+/**
+ * Converts a sentry UUID to a string representation used for internal
+ * sentry UUIDs such as event IDs.
+ */
+void sentry__internal_uuid_as_string(const sentry_uuid_t *uuid, char str[37]);
+
+/**
+ * Converts a sentry UUID to a string representation used for span IDs.
+ */
+void sentry__span_uuid_as_string(const sentry_uuid_t *uuid, char str[17]);
+
 #ifdef SENTRY_PLATFORM_WINDOWS
 /**
  * Create a new UUID from the windows-native GUID type.
  */
-sentry_uuid_t sentry__uuid_from_native(GUID *guid);
+sentry_uuid_t sentry__uuid_from_native(const GUID *guid);
```

Comment:
looks like including this in `sentry_uuid.c` revealed a mismatch between this signature and the one in `sentry_uuid.c`, causing the windows builds to fail.

unsure if the new include subtly breaks anything; hoping it doesn't.

Reply:
This shouldn’t cause any problems.